exercises/exercise-3.js

```diff
@@ -7,8 +7,13 @@
   Considerations: Values can be nested and datypes Number, String or Object.
 */
 
-function deepStringSum() {
-  // TODO: implement
+function deepStringSum(obj) {
+  let counter = 0;
+  for (var i in obj) {
```

Comment:
`var i`could give you unexpected behaviours if you don't take care of it, i recommend you to use `const i` if you don't have to live worried 😅